src/rules/no-default-export.js

```diff
@@ -20,19 +20,20 @@ module.exports = {
       `Do not alias \`${local.name}\` as \`default\`. Just export ` +
       `\`${local.name}\` itself instead.`;
 
+
     return {
       ExportDefaultDeclaration(node) {
-        context.report({ node, message: preferNamed });
+        context.report({ node, message: preferNamed, loc: context.getSourceCode().getFirstTokens(node, 2)[1].loc });
```

Comment:
The first tokens are "export" and "default". We want to highlight the loc[ation] of the "default" token.